eos-enable-zram

```diff
@@ -54,6 +54,17 @@ if [ -z "${root_disk}" ]; then
     exit 0
 fi
 
+for swap_area in "${active_swaps[@]}" ; do
+    if [ "${swap_area%?}" = "${root_disk}" ] ; then
```

Comment:
Does this work in the `${root_disk}p${partnum}` case?

Could we write a test for this script?

Reply:
I does not, good catch. And I'll think of a testing strategy for this.

Reply:
FWIW the strategy I used for `eos-repartition.sh` is to make it accept arguments on the command line so it operates on files/loopback devices controlled by the test case; with the real values being used if the command line is empty. But in this case the script also looks at global system state so it may be a bit harder.